From: Lee Duncan <lduncan@suse.com>
Date: Sat May  6 12:20:15 PDT 2017
Subject: [PATCH] kABI: move and hide new cxgbi device owner field
References: bsc#1018885
Patch-mainline: Never, kabi

Move the new 'owner' field in struct cxgbi_device to the end
of the struct and hide it for the kABI checker.

Signed-off-by: Lee Duncan <lduncan@suse.com>
---
diff a/drivers/scsi/cxgbi/libcxgbi.h b/drivers/scsi/cxgbi/libcxgbi.h
---
 drivers/scsi/cxgbi/libcxgbi.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/drivers/scsi/cxgbi/libcxgbi.h
+++ b/drivers/scsi/cxgbi/libcxgbi.h
@@ -540,7 +540,6 @@ struct cxgbi_device {
 	struct pci_dev *pdev;
 	struct dentry *debugfs_root;
 	struct iscsi_transport *itp;
-	struct module *owner;
 
 	unsigned int pfvf;
 	unsigned int rx_credit_thres;
@@ -573,6 +572,9 @@ struct cxgbi_device {
 	int (*csk_init_act_open)(struct cxgbi_sock *);
 
 	void *dd_data;
+#ifndef	__GENKSYMS__
+	struct module *owner;
+#endif
 };
 #define cxgbi_cdev_priv(cdev)	((cdev)->dd_data)
 
